example/kernelSpecialization/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2014-2021 Erik Zenker, Benjamin Worpitz, Jan Stephan, Sergei Bastrakov
+#
+# This file exemplifies usage of alpaka.
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED “AS IS” AND ISC DISCLAIMS ALL WARRANTIES WITH
+# REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL ISC BE LIABLE FOR ANY
+# SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
+# IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+################################################################################
+# Required CMake version.
+
+cmake_minimum_required(VERSION 3.15)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+################################################################################
+# Project.
+
+set(_TARGET_NAME kernelSpecialization)
+
+project(${_TARGET_NAME})
```

Comment:
I always wondered why we define our own variable, when we can just use `${PROJECT_NAME}` after the call to `project(...)`.

Reply:
I do not know. When you compile this example within alpaka is `${PROJECT_NAME}` then `alpaka` or the name of this example?

Reply:
Name of example.